code/src/App.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import tasks from 'reducers/tasks'
+import OverView from 'components/OverView'
+
+const reducer = combineReducers({
+  tasks: tasks.reducer,
+})
+
+const store = configureStore({
+  reducer,
+})
+
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <OverView />
```

Comment:
Nice wrapping all the other components inside a parent component. Love this approach! 👍 ✨